components/feature/search/src/main/assets/search/list.json

```diff
@@ -631,7 +659,7 @@
     "pl": {
       "default": {
         "visibleDefaultEngines": [
-          "google-b-m", "bing", "ddg", "wikipedia-pl"
+          "google-b-m", "bing", "ddg", "wikipedia-pl", "ebay-pl"
```

Comment:
Desktop doesn't currently ship ebay.pl. We should double-check the requirement here.

Reply:
Sure, I'll double check this.

Reply:
Confirmed that we'll keep `eBay.pl`